tests/data_examples/metadata/update-root-payload.json

```diff
@@ -1,4 +1,18 @@
 {
+  "settings": {
+    "expiration": {
+      "root": 365,
+      "targets": 365,
+      "snapshot": 1,
+      "timestamp": 1,
+      "bins": 1
+    },
+    "services": {
+      "targets_base_url": "http://www.example.com/repository/",
+      "number_of_delegated_bins": 4,
+      "targets_online_key": true
+    }
```

Comment:
Can we remove the `services` field as we decided we don't want to support it for the moment?
We need to also update the `MetadataPostPayload` to include `settings` and `expiration`:
https://github.com/repository-service-tuf/repository-service-tuf-api/blob/8c22914067d5fa65cc342cce0259e72549018252/repository_service_tuf_api/metadata.py#L34

Reply:
I will do it in another PR.

Reply:
I filed another issue and signed to myself.
- https://github.com/repository-service-tuf/repository-service-tuf-api/issues/340

Reply:
I added a new commit. :)